CRM/Utils/VersionCheck.php

```diff
@@ -382,6 +384,33 @@ private function getExtensionStats() {
   }
 
   /**
+   * Fetch stats about domain and add to 'stats' array.
+   */
+  private function getDomainStats() {
+    $params = array(
+      'id' => CRM_Core_Config::domainID(),
+    );
+    $domain_result = civicrm_api3('domain', 'getsingle', $params);
+    if (!empty($domain_result['contact_id'])) {
+      $address_params = array(
+        'contact_id' => $domain_result['contact_id'],
+        'is_primary' => 1,
+        'sequential' => 1,
+      );
+      $address_result = civicrm_api3('address', 'get', $address_params);
+      if ($address_result['count'] == 1) {
+        $country_params = array(
+          'id' => $address_result['values'][0]['country_id'],
+        );
+        $country_result = civicrm_api3('country', 'getsingle', $country_params);
+        if (!empty($country_result['iso_code'])) {
+          $this->stats['domain_country_iso'] = $country_result['iso_code'];
```

Comment:
Can you please rename 'domain_country_iso' to 'domain_isoCode' for consistency with server-side code? Sorry and thanks.